system/ee/ExpressionEngine/Model/Status/Status.php

```diff
@@ -141,6 +141,8 @@ public function getSelectOptionComponent($use_ids = false)
 
     public function renderTag()
     {
+        ee()->lang->loadfile('admin_content');
```

Comment:
Without this, the status tag is rendered as `open` and `closed`, not `Open` and `Closed`.